Replace symlink with an actual copy of the env file in the kube base (#194)
faros-ai/faros-community-edition: kube/README.md

Provided you have have access to a Kubernetes cluster and have configured access to it via `kubectl`, FarosCE can be deployed with the following command:

```base
kubectl apply -k https://github.com/faros-ai/faros-community-edition/kube/base
```

> Kustomize will load the deployment definition in this repository over Github, build the manifests and pipe them over to `kubectl` for deployment in the configured cluster.

Reviewer: ```suggestion
Kustomize will load the deployment definition in this repository over Github, build the manifests and deploy in the configured cluster.
```
